sdk/helper/pluginutil/multiplexing.go

```diff
@@ -9,6 +9,8 @@ import (
 	status "google.golang.org/grpc/status"
 )
 
+const MultiplexingCtxKey string = "multiplex_id"
```

Comment:
Just moved this from [runner.go](https://github.com/hashicorp/vault/pull/15671/files#diff-38bd55854da0804411382307ed24da71938ae837c40f38d495a467b05535f831L41) as it seemed to belong here instead.